src/pages/osobne-udaje.tsx

```diff
@@ -201,10 +202,6 @@ export const validate = (values: PersonalInformationUserInput) => {
     errors.r004_priezvisko = 'Zadajte vaše priezvisko'
   }
 
-  if (!values.r007_ulica) {
-    errors.r007_ulica = 'Zadajte ulicu'
```

Comment:
Neviem ci je to az take stastne riesenie. Co ked len zabudne niekto zadat ulicu a bude tam potom mat napisane "Bratislava"? Nie su ludia ktori byvaju v obciach bez ulic aj tak zvyknuti pisat obec do ulice?

Reply:
Hej. Nebolo by lepsie dat ulicu jednoducho nepovinne? Taketo overly smart automaticke doplnanie moze byt casto pre usera nezrozumitelne a frustrujuce, lebo netusim ze co a preco sa deje

Reply:
Vyhodil som ten fallback.
Aj ked z mojej skusenosti vacsina ludi vie ze ked nema ulicu tak tam automaticky napise nazov obce, takze mne sa ten fallback zda ok. (alebo aj nechat to pole povinne je ok)

Inac mozes mat take situacie ked mas adresu:

```
Ulica 50
Obec
```

Tak na riadku kde je ulica bude len same cislo a to vyzera divne.

Reply:
Pozeram [formular](https://priznanie.staging.slovensko.digital/form/form.451.html) ze ulica nie je povinna. Ale mohli by sme upravit aspon jeden use case v `cases.test.ts` aby tiez nemal ulicu (resp aby mal prazdy string). Pridavat novy use case len kvoli chybajucej ulici mi pride zbytocne.